extensions/common/vault/vault-hashicorp/src/main/java/org/eclipse/edc/vault/hashicorp/HashicorpVaultExtension.java

```diff
@@ -71,6 +71,9 @@ public class HashicorpVaultExtension implements ServiceExtension {
     @Setting(value = "The URL path of the vault's /secret endpoint", defaultValue = VAULT_API_SECRET_PATH_DEFAULT)
     public static final String VAULT_API_SECRET_PATH = "edc.vault.hashicorp.api.secret.path";
 
+    @Setting(value = "The path of the folder that the secret is stored in", required = false, defaultValue = "")
```

Comment:
```suggestion
    @Setting(value = "The path to the folder in which all the secrets of/for the EDC should be stored in", required = false, defaultValue = "")
```

Reply:
FWIW, the original is correct and more concise